tutorials/plot_object_annotations.py

```diff
@@ -75,19 +78,25 @@
 mne.viz.plot_events(events, raw.info['sfreq'], raw.first_samp, color=color,
                     event_id=event_id)
 
-# Create some annotations specifying onset, duration and description
+###############################################################################
+# Next, we're going to find instances of eye blinks and turn them into
+# :class:`Annotations <mne.Annotations>`:
+
+# find blinks
 annotated_blink_raw = raw.copy()
 eog_events = mne.preprocessing.find_eog_events(raw)
 n_blinks = len(eog_events)
-# Center to cover the whole blink with full duration of 0.5s:
+
+# Turn blink events into Annotations of 0.5 seconds duration,
+# each centered on the blink event:
 onset = eog_events[:, 0] / raw.info['sfreq'] - 0.25
 duration = np.repeat(0.5, n_blinks)
 description = ['bad blink'] * n_blinks
-annot = mne.Annotations(onset, duration, description,
-                        orig_time=raw.info['meas_date'])
```

Comment:
why removing orig_time? it's not equivalent. If you don't give orig_time they you ignore the raw.first_samp

Reply:
Sorry, I thought that it is equivalent.